--- xerces-2_11_0/tools/org/apache/xerces/util/XJavac.java.libgcj	2013-01-25 01:30:27.551651045 +0100
+++ xerces-2_11_0/tools/org/apache/xerces/util/XJavac.java	2013-01-25 01:31:34.236841080 +0100
@@ -70,7 +70,8 @@ public class XJavac extends Javac {
                      (vendor.indexOf("HEWLETT-PACKARD") >= 0) ||
                      (vendor.indexOf("KAFFE") >= 0) ||
                      (vendor.indexOf("SABLE") >= 0) ||
-                     (vendor.indexOf("FREEBSD") >= 0)) {
+                     (vendor.indexOf("FREEBSD") >= 0) ||
+		     (vendor.indexOf("FREE SOFTWARE FOUNDATION") >= 0)) {
                 // we're on an SUN 1.4 or higher; fiddle with the bootclasspath.
                 // since we can't eviscerate XML-related info here,
                 // we must use the classpath
